### PR TITLE
Aqara wall outlet H2

### PR DESCRIPTION
**Pull request by kirovilya**
https://aqara.ru/product/%D1%83%D0%BC%D0%BD%D0%B0%D1%8F-%D0%B2%D1%81%D1%82%D1%80%D0%B0%D0%B8%D0%B2%D0%B0%D0%B5%D0%BC%D0%B0%D1%8F-zigbee-%D1%80%D0%BE%D0%B7%D0%B5%D1%82%D0%BA%D0%B0-aqara-wall-outlet-h2-eu/

### CODE REVIEW

File: src/lib/xiaomi.ts [hunk 1]
```diff
@@ -1468,6 +1468,68 @@ export const xiaomiModernExtend = {
...
+        description: 'Maximum allowed load, turns off if exceeded',
+        valueMin: 100,
+        valueMax: 3840,
+        unit: 'W',
+        access: 'ALL',
+        zigbeeCommandOptions: {manufacturerCode},
+        ...args,
+    }),
+    xiaomiDoNotDisturb: (args? :Partial<modernExtend.BinaryArgs>) => modernExtend.binary({
+        name: 'do_not_disturb',
```
Comment (Koenkk) [inline src/lib/xiaomi.ts:1512]: I propose to call this `power_outage_memory`

Comment (kirovilya) [reply src/lib/xiaomi.ts:1512]: no, it's turning off the indicator on the outlet :)
here is a similar converter https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/converters/toZigbee.ts#L2512

Comment (Koenkk) [reply src/lib/xiaomi.ts:1512]: Maybe we should call it `led_indicator` then?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**